swagger_server/itm/itm_ta1_controller.py

```diff
@@ -5,16 +5,21 @@
 from swagger_server.models.probe_response import ProbeResponse  # noqa: F401,E501
 from swagger_server.models.alignment_results import AlignmentResults  # noqa: F401,E501
 from swagger_server.models.alignment_target import AlignmentTarget  # noqa: F401,E501
+from swagger_server import config_util
 
-class ITMTa1Controller:
-    ADEPT_PORT = os.getenv("ADEPT_PORT")
-    if (ADEPT_PORT is None or ADEPT_PORT == ""):
-        ADEPT_PORT = '8081'
-    SOARTECH_PORT = os.getenv("SOARTECH_PORT")
-    if (SOARTECH_PORT is None or SOARTECH_PORT == ""):
-        SOARTECH_PORT = '8084'
 
-    def __init__(self, alignment_target_id, scene_type, alignment_target = None):
+class ITMTa1Controller:
+    
+    config_util.check_ini()
+    config = config_util.read_ini()[0]
+    
+    
+    ADEPT_PORT = config['DEFAULT']['ADEPT_PORT']
+    SOARTECH_PORT = config['DEFAULT']['SOARTECH_PORT']
+    ADEPT_HOSTNAME = config['DEFAULT']['ADEPT_HOSTNAME']
+    SOARTECH_HOSTNAME = config["DEFAULT"]["SOARTECH_HOSTNAME"]
+    
```

Comment:
As discussed elsewhere, change to `ADEPT_URL` and `SOARTECH_URL` and update the rest of the code below.